src/brevitas/graph/calibrate.py

```diff
@@ -85,11 +89,33 @@ def __exit__(self, type, value, traceback):
             self.model, is_training=self.previous_training_state, quantization_enabled=True)
 
 
+class allow_unexpected_bias_keys:
+
+    def __init__(self, model):
+        self.model = model
+
+    def __enter__(self):
+        self.tracked_modules = []
+        for module in self.model.modules():
+            if issubclass(type(module), QuantWBIOL):
+                if module.bias is None:
+                    module.register_parameter(
+                        'bias',
+                        nn.Parameter(torch.empty(module.weight.shape[0])).to(module.weight.device))
+                    self.tracked_modules.append(module)
+
+    def __exit__(self, type, value, traceback):
+        for module in self.tracked_modules:
+            # empty tensor has a numel result of 0
+            if torch.numel(module.bias) == 0:
```

Comment:
let's check that str(module.device) == 'meta'

Reply:
the approach i've used doesn't use the meta device but rather an empty tensor as the meta tensor seemed to add a lot of complication and warnings

Reply:
If I'm not mistaken, an empty tensor has "meta" as device by default. Or am I missing something?

Reply:
My testing has the default as 'cpu'
`torch.empty(10).device` 
gives 'cpu'

Reply:
Out of curiosity, what warnings do you get with meta tensors?

Reply:
I didn't go too deep into a meta tensor implementation but warnings like:
> UserWarning: for net.0.bias: copying from a non-meta parameter in the checkpoint to a meta parameter in the current model, which is a no-op.